x/incentive/types/keys.go

```diff
@@ -40,3 +42,42 @@ var (
 	EarnRewardIndexesKeyPrefix                    = []byte{0x19} // prefix for key that stores earn reward indexes
 	PreviousEarnRewardAccrualTimeKeyPrefix        = []byte{0x20} // prefix for key that stores the previous time earn rewards accrued
 )
+
+var (
+	ClaimKeyPrefix                     = []byte{0x21}
+	RewardIndexesKeyPrefix             = []byte{0x22}
+	PreviousRewardAccrualTimeKeyPrefix = []byte{0x23}
+)
+
+var sep = []byte("|")
+
+func createKey(bytes ...[]byte) (r []byte) {
+	for _, b := range bytes {
+		r = append(r, b...)
+	}
+	return
+}
+
+func getKeyPrefix(dataTypePrefix []byte, claimType ClaimType) []byte {
+	b := make([]byte, 4)
+	binary.LittleEndian.PutUint32(b, uint32(claimType))
```

Comment:
any reason for little endian over big endian?

Reply:
hmm no reason in particular, would there be a benefit for one over the other?

Reply:
In this case I think there's no difference. But if we were creating an key for `int32`s it would create an odd sort order as the last byte would be first in the key. Also casting `claimType` (`int32` underlying) to `uint32` orders negative numbers after positive.

I'd probably standardize on big endian, just to avoid our future selves using it when a sort order is needed.

Reply:
ah right sort order is a good point, will update it accordingly